schemas/SlashCommand.json

```diff
@@ -6,60 +6,8 @@
       "enum": ["SlashCommand"]
     },
     "command": {
-      "description": "The name of this command.",
-      "type": "string",
-      "enum": [
-        "Spoiler",
-        "Shrug",
-        "TableFlip",
-        "Unflip",
-        "Lenny",
-        "Plain",
-        "Html",
-        "UpgradeRoom",
-        "JumpToDate",
-        "Nick",
-        "MyRoomNick",
-        "RoomAvatar",
-        "MyRoomAvatar",
-        "MyAvatar",
-        "Topic",
-        "RoomName",
-        "Invite",
-        "Join",
-        "Part",
-        "Remove",
-        "Ban",
-        "Unban",
-        "Ignore",
-        "Unignore",
-        "Op",
-        "Deop",
-        "DevTools",
-        "AddWidget",
-        "Verify",
-        "DiscardSession",
-        "RemakeOlm",
-        "Rainbow",
-        "RainbowMe",
-        "Help",
-        "Whois",
-        "Rageshake",
-        "ToVirtual",
-        "Query",
-        "Msg",
-        "HoldCall",
-        "UnholdCall",
-        "ConvertToDM",
-        "ConvertToRoom",
-        "Me",
-        "Confetti",
-        "Fireworks",
-        "Hearts",
-        "Rainfall",
-        "Snowfall",
-        "SpaceInvaders"
-      ]
+      "description": "The the slash command text. e.g. /me ",
```

Comment:
Does this make sense @t3chguy ?

So you might nee update any reports you have that use the old format(`Part`/`Invite`)?

Reply:
Hm good point, losing historical data would not be good. Not sure what the best solution is here then.